rvv-intrinsic-rfc.md

```diff
@@ -609,6 +610,25 @@ Example:
 vint64m1_t vreinterpret_v_i32m1_i64m1(vint32m1_t src)
 ```
 
+### LMUL truncation and LMUL extension functions<a name="lmul-trunc)"></a>
+
+These utility functions help users to truncate or extent current LMUL under same SEW regardless of vl (keep current vl).
```

Comment:
Do you mind having more description on `LMUL extension`? Maybe `The value for extension part of variable will be undefined value.`?

> same SEW regardless of vl (keep current vl).
I assume that means `vl` register won't affected/changed? if so, maybe ` (keep current vl).` ->  `, and won't change content of vl register `?